src/routes/docs/products/ai/integrations/togetherai/+page.markdoc

```diff
@@ -0,0 +1,326 @@
+      <div class="card u-flex u-gap-24 u-flex-vertical">
+        <div class="u-flex u-cross-center u-gap-8">
+          <div class="input-text-wrapper is-with-end-button u-width-full-line">
+            <input x-model="prompt" type="search" placeholder="Prompt" />
+            <div class="icon-search" aria-hidden="true"></div>
+          </div>
+          <div class="select u-width-140">
+            <select x-model="type">
+              <option value="text">Text</option>
+              <option value="image">Image</option>
+            </select>
+            <span class="icon-cheveron-down" aria-hidden="true"></span>
+          </div>
+
+          <button class="button" x-bind:disabled="loading"
+            x-on:click="async () => { loading = true; answer = {type: '', answer: ''}; try { answer = await onSubmit(prompt, type) } catch(err) { console.error(err); } finally { loading = false; } }">
+            <span class="text">Generate</span>
+          </button>
+        </div>
+        <template x-if="answer.type">
+          <div class="u-flex u-flex-vertical u-gap-12">
+            <div class="u-flex u-flex-vertical u-gap-12 card">
+              <div class="u-flex u-gap-12">
+                <h5 class="eyebrow-heading-2">Result:</h5>
+              </div>
+              <template x-if="answer.type === 'image'" class="u-flex u-gap-12">
+                <img class="u-max-width-400" x-bind:src="answer.response" alt="Together output" />
+              </template>
+              <template x-if="answer.type === 'text'" class="u-flex u-gap-12">
+                <p class="u-color-text-gray" x-text="answer.response"></p>
+              </template>
+            </div>
+          </div>
+        </template>
+      </div>
+    </div>
+  </main>
+</body>
+```
+
+Together, this will render a form to submit your question to the Appwrite Function through a POST request, which you'll create next. The Appwrite Function will call Together AI's API and return the response, which will be displayed on your page using different conditional statements depending on the output media type.
+{% /section %}
+
+{% section #step-6 step=6 title="Handle POST Request" %}
+Now that you're serving a basic HTML page add methods necessary to integrate with Together AI's API.
+
+Import `fetch` from undici and the Appwrite SDK
+
+```js
+import { fetch } from 'undici'
+import { Client, ID, InputFile, Storage } from 'node-appwrite';
+```
+
+Next, after you serve the HTML, add code to validate the body of the request and define the models:
+
+```js
+const models = {
+  'text': 'mistralai/Mixtral-8x7B-Instruct-v0.1',
+  'image': 'stabilityai/stable-diffusion-xl-base-1.0'
+};
+
+if (!req.body.prompt || typeof req.body.prompt !== 'string') {
+  return res.json({ ok: false, error: 'Missing required field `prompt`' }, 400);
+}
+
+if (req.body.type !== 'text' && req.body.type !== 'image') {
+  return res.json({ ok: false, error: 'Invalid field `type`' }, 400);
+}
+```
```

Comment:
There is no indication where this code needs to be added. You need to do something like this 

```js
export default async ({ req, res, error }) => {
    // Code for GET request handler ...
   Your code here. 
};
```